src/rad/resources/schemas/visit-1.1.0.yaml

```
@@ -0,0 +1,153 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/visit-1.1.0
+
+title: Visit Information
+type: object
+properties:
+  dither:
+    title: Dither Pattern Information
+    type: object
+    properties:
+      primary_name:
+        title: Primary Dither Pattern Name
+        description: |
+          Name of the primary dither pattern used for
+          the visit.
+        anyOf:
+          - type: string
+          - type: "null"
+        sdf:
+          special_processing: VALUE_REQUIRED
+          source:
+            origin: TBD
```

```suggestion
            origin: PSS:dms_visit.dither_type
```
Lost change from https://github.com/spacetelescope/rad/pull/571